skript.js

```diff
@@ -0,0 +1,55 @@
+
+GETphoto("https://randomuser.me/api/?results=50")
+.then(
+  response=>alert("sucsess"),
+  error=>alert(`Rejected: ${error}`)
+);
+
+window.addEventListener("scroll", function(){
+    let block = document.getElementById('photo-area');
```

Comment:
Use const instead of let everywhere where you can avoid let. It helps to understand variable type during the execution